majority-multisign/test/Spec/Instances

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Should this file still be here?

Reply:
Only until I start using `quickcheck-plutus-instances`, but I'll need to upgrade Plutus before that and I wanted to merge this branch before that step.

Reply:
I don't follow, this isn't even a `hs` file, is this a ghc feature _don't know abnout?

Reply:
Oh sorry, I thought you were asking about `Instances.hs` and I didn't realize I had committed a blank file. I've removed it now.